app/views/users/show.html.erb

```diff
@@ -15,6 +15,7 @@
             </li>
             <li>
               <%= link_to t(".my notes"), user_notes_path(@user) %>
+              <span class='badge count-number'><%= number_with_delimiter(current_user.notes.size) %></span>
```

Comment:
This should be '@user', should it?

Reply:
Dear @tordans ,

Thank you for reviewing. In this case, I think they are same (2 proxies to the same object). Maybe I'm wrong :-) ?

Thanks,
Nenad.

Reply:
They are not generally the same. `current_user` is the currently logged in user, and `@user` is the user whose profile page you are showing.

Now in this specific situation, they are the same user, since they come after an equality check (`if current_user and @user.id == current_user.id`), so they can be used interchangeable - but elsewhere in this file they cannot.

I think the problem here is that although this block is mostly using `current_user` (e.g. for the edits count, 6 lines above), the line immediately above is using `@user`, and so seeing it switch between two lines suggests an error. My initial reaction is that we should use `current_user` throughout the `<!-- Displaying user's own profile page -->` section, for clarity.